examples/01-basic/11-placeholder/App.tsx

```diff
@@ -0,0 +1,26 @@
+import "@blocknote/core/fonts/inter.css";
+import { BlockNoteView } from "@blocknote/mantine";
+import "@blocknote/mantine/style.css";
+import { useCreateBlockNote } from "@blocknote/react";
+import { locales } from "@blocknote/core";
+
+export default function App() {
+  // Set the user locale.
+  const userLocale = "en";
+
+  const placeholder = "This is a custom placeholder";
+
+  // Creates a new editor instance.
+  const editor = useCreateBlockNote({
+    dictionary: {
+      ...locales[userLocale],
+      placeholders: {
+        ...locales[userLocale].placeholders,
+        default: placeholder ?? locales[userLocale].placeholders.default,
```

Comment:
just pass in "This is a custom placeholder" directly, and remove the `placeholder` variable